ACE/ace/CDR_Base.cpp

```diff
@@ -948,8 +951,8 @@ void ACE_CDR::Fixed::normalize (UShort min_scale)
 
 ACE_CDR::Fixed ACE_CDR::Fixed::from_string (const char *str)
 {
-  const bool negative = str && *str == '-';
-  if (negative || (str && *str == '+'))
+  const bool negative = *str == '-';
```

Comment:
I went back to coverity's web interface and I couldn't see any outstanding issues in CDR_Base.  I'm going to consider this one closed unless I hear otherwise.